build/ci_utils/src/cache/goodie/flatc.rs

```diff
@@ -0,0 +1,82 @@
+use crate::prelude::*;
+
+use crate::cache::goodie;
+use crate::cache::Cache;
+use crate::env::known::PATH;
+use crate::env::Modification;
+use crate::github::RepoRef;
+
+
+
+/// The repository of the FlatBuffers project.
+pub const REPO: RepoRef = RepoRef { owner: "google", name: "flatbuffers" };
+
+#[derive(Clone, Debug)]
+pub struct Flatc {
+    /// The version of the `flatc` binary.
+    pub version:  Version,
+    /// Operating system for which the binary is built.
+    pub platform: OS,
+}
+
+impl Flatc {
+    /// Get the download URL.
+    ///
+    /// The package is a zip file containing the `flatc` binary.
+    ///
+    /// ```
+    /// use ide_ci::prelude::*;
+    /// use ide_ci::cache::goodie::flatc::Flatc;
+    ///
+    /// # fn main() -> Result {
+    /// let version = Version::from_str("24.3.25")?;
+    /// let flatc = Flatc { version, platform: OS::Linux };
+    /// let url = flatc.url()?;
+    /// assert_eq!(url.as_str(), "https://github.com/google/flatbuffers/releases/download/v24.3.25/Linux.flatc.binary.clang++-15.zip");
+    /// # Ok(())
+    /// # }
+    pub fn url(&self) -> Result<Url> {
+        let filename = match (&self.version, self.platform) {
+            (version, OS::Windows) if version >= &Version::new(2, 0, 0) =>
```

Comment:
We do support multiple versions, but the only one used is `24.3.25`, right?

Reply:
Yes. The Rust code for downloading the `flatc` binary supports all versions `>=2.0.0`, however the only one used is the one defined in `build.sbt`.
I used this previously for comparing the generated code between different `flatc` versions (when investigating bumping it for GUI as well) and thought there is no harm in keeping extra bit of leeway here.